prisma/schema.prisma

```diff
@@ -0,0 +1,122 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("DPDMONGO_DATABASE_URL")
+}
+
+type ConfigsConfig {
+  /// This field was commented out because of an invalid name. Please provide a valid one that matches [a-zA-Z][a-zA-Z0-9_]*
+  // 0 ConfigsConfig0[] @map("0")
+  colormap ConfigsConfigColormap[]
+}
+
+type ConfigsConfig0 {
+  id       Int      @map("_id")
+  analysis String
+  category String
+  color    String[]
+  label    String
+  range    Int[]
+  text     Boolean
+  variable String
+}
+
+type ConfigsConfigColormap {
+  analysis String
+  category String
+  color    String[]
+  label    String
+  range    Int[]
+  text     Boolean
+  variable String
+}
+
+type ReportsCharts {
```

Comment:
this looks very similar to the charts we created. Are they intended to be the same?